### PR TITLE
📝 Update install instructions around known error

### LINKED ISSUE #109: Failed to open 'v8.pc': No such file or directory
```
go get -u github.com/ry/deno/...
# pkg-config --cflags v8.pc
Failed to open 'v8.pc': No such file or directory
No package 'v8.pc' found
pkg-config: exit status 1
```

This looked similar to https://github.com/ry/v8worker/issues/23 however the fix recommended there did not fix the problem at hand.

### PR DESCRIPTION
**Pull request by samccone**
Even though the readme told me there was going to be an error, I was still confused about this error, as it looked unrelated to fetching the sources.

To clarify this update the readme to spell out the expected error.

Fixes #109

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,9 +92,16 @@ go get -u github.com/golang/protobuf/protoc-gen-go
 go get -u github.com/jteeuwen/go-bindata/...
 ```
 
-You need to get and build [v8worker2](https://github.com/ry/v8worker2).  The package will not build with `go
-get` and will log out an error, which can be ignored. It takes about 30 minutes
-to build:
+You need to get and build [v8worker2](https://github.com/ry/v8worker2).  __The package will not build with `go
+get` and will log out an error ⚠__
+```
```
Comment (yorkie) [inline README.md]: I think add a `bash` to keep consistent with others would be better :)

**Comment by samccone** [comment]
Updated, thank you for taking the time to look and provide feedback.

File: README.md [hunk 2]
```diff
@@ -92,9 +92,16 @@ go get -u github.com/golang/protobuf/protoc-gen-go
...
+You need to get and build [v8worker2](https://github.com/ry/v8worker2).  __The package will not build with `go
+get` and will log out an error ⚠__
+```bash
+# pkg-config --cflags v8.pc
+Failed to open 'v8.pc': No such file or directory
+No package 'v8.pc' found
+pkg-config: exit status 1
+```
+
+, __which can be ignored__. It takes about 30 minutes to build:
```
Comment (yorkie) [inline README.md]: BTW, i think the ","(comma symbol) are not needed as well.

Comment (samccone) [reply README.md]: Sounds good to me, thanks for taking such a close look.

**Comment by ry** [comment]
Sure. Thanks.

**Pull request merged**